tests/unit/test_charm.py

```diff
@@ -699,16 +707,51 @@ def test_ingress(
 
     assert charm.ingress.config_dict == {
         "service-hostname": app_name,
+        "host": app_name,
         "service-name": app_name,
+        "name": app_name,
         "service-port": "80",
+        "port": "80",
+        "owasp-modsecurity-crs": True,
+        "owasp-modsecurity-custom-rules": 'SecAction "id:900130,phase:1,nolog,pass,t:none,setvar:tx.crs_exclusions_wordpress=1"\n',
     }
 
     harness.update_config({"tls_secret_name": "tls_secret"})
 
     assert charm.ingress.config_dict == {
         "service-hostname": app_name,
+        "host": app_name,
+        "service-name": app_name,
+        "name": app_name,
+        "service-port": "80",
+        "port": "80",
+        "tls-secret-name": "tls_secret",
+        "owasp-modsecurity-crs": True,
+        "owasp-modsecurity-custom-rules": 'SecAction "id:900130,phase:1,nolog,pass,t:none,setvar:tx.crs_exclusions_wordpress=1"\n',
```

Comment:
Out of curiosity, how is `id:900130` generated?

Reply:
It the rule id to disable certain checks for some well-known applications.
https://github.com/SpiderLabs/owasp-modsecurity-crs/blob/56ff01a8408bd387fe14f7856391c48e245b7937/crs-setup.conf.example#L321-L362